lib/sedna/core/joint_inference/joint_inference.py

```diff
@@ -32,13 +32,22 @@ class TSBigModelService(JobBase):
     Provides RESTful interfaces for large-model inference.
     """
 
-    def __init__(self, estimator=None, config=None):
-        super(TSBigModelService, self).__init__(
-            estimator=estimator, config=config)
+    def __init__(self, estimator=None):
+        """
+        Initial a big model service for JointInference
+        :param estimator: Customize estimator
+        """
+
+        super(TSBigModelService, self).__init__(estimator=estimator)
         self.local_ip = self.get_parameters("BIG_MODEL_BIND_IP", get_host_ip())
         self.port = int(self.get_parameters("BIG_MODEL_BIND_PORT", "5000"))
 
```

Comment:
these constant may put in `constant.py`

Reply:
Generally, the value is obtained from the environment variable delivered by the CRD.